Moli/Core/Screens/Home/Live/MoliRoomsSnapCarousel.swift

```diff
@@ -0,0 +1,167 @@
+
+import SwiftUI
+
+struct MoliRoomsSnapCarousel: View
+{
+    var UIState: UIStateModel
+    var spacing: CGFloat = 8
+    var widthOfHiddenCards: CGFloat = 8
+    var cardHeight: CGFloat = 180
```

Comment:
:art: Om parametrerna inte används utanför dess komponent så är det alltid bra att hålla de till en private värde så att man kan göra koden mer typsäker, sen kan man ändra till en `let` om man vill, personligen tycker jag det hjälper att förstå vad som händer eftersom jag vet att värdet inte kommer att ändras
 
```suggestion
    private let spacing: CGFloat = 8
    private let widthOfHiddenCards: CGFloat = 8
    private let cardHeight: CGFloat = 180
```